ee/src/main/java/org/jboss/as/ee/component/ViewDescription.java

```diff
@@ -179,12 +179,25 @@ public Deque<ViewConfigurator> getConfigurators() {
      * Create the injection source
      *
      * @param serviceName     The view service name
-     * @param viewClassLoader
+     * @param viewClassLoader The view class loader
+     * @param appclient       appclient environment
+     * @deprecated use {@link ViewDescription#createInjectionSource(ServiceName, Supplier, boolean)} instead. This method will be removed in the future.
      */
     protected InjectionSource createInjectionSource(final ServiceName serviceName, Value<ClassLoader> viewClassLoader, boolean appclient) {
         return new ViewBindingInjectionSource(serviceName);
     }
```

Comment:
any reason this method is not removed, or the Value not converted to Supplier? It's overloaded with the next one. A quick check show no usage of this protected method.

Reply:
I removed this BC method. You are right it is not necessary. Only code in WildFly used it.